webextensions/javascript-apis.json

```diff
@@ -5530,7 +5530,10 @@
                     "version_added": "26"
                   },
                   "edge": {
-                    "version_added": true
+                    "version_added": true,
+                    "notes": [
+                      "The `tab` object is incomplete."
```

Comment:
This isn't super-helpful, but neither is the [note on Microsoft's site](https://docs.microsoft.com/en-us/microsoft-edge/extensions/api-support/supported-apis#runtime):

> tab object in runtime.onMessage event is not fully implemented.

@abbycar , do you have any more detail on this?